src/inputs/dropdown/dropdown.jsx

```diff
@@ -670,4 +715,8 @@ Dropdown.propTypes = {
     ]),
 };
 
+Dropdown.defaultProps = {
+    autoScrollSelection: true,
```

Comment:
Since this is a new behavior, are we sure we want to default this to `true`?  Is the thinking that this is, in general, desirable behavior that should be the default, and opted out of by exception, versus having to opt into it everywhere?

Reply:
How many days do we have to regression test 1.29.0 @groberts314?

Reply:
So, here was the originally proposed schedule:
* Dev was supposed to be feature-complete by EOD _yesterday_ Thurs Sept 24
* QA is supposed to complete its initial feature-level QA by EOD Tues Sept 29
* Release Branching and HC-Stage deployment of initial 1.29.0 RC builds is supposed to happen on Wed Sept 30
* QA does regression testing thru EOD Wed Oct 7
* Production release on Thurs Oct 8

Reply:
I am definitely more comfortable with defaulting this to `false`.

We'll need to ensure this is turned on in the relevant placement in the UI in HC, once the updated React CM UI is available.  Hopefully we can figure out what that is from the ticket and other HC UI changes, or perhaps Tatiana can help us identify it if we're not sure.  Unless we're going to kick this back to @IlyaRadinsky to handle and we review again on Monday, though that will jeopardize the schedule a bit.